tcf_website/templates/base/base.html

```diff
@@ -69,6 +69,7 @@
     return "";
   }
   // Update copyright year in the sidebar
+  // Adding a comment to test if deployment succeeded
```

Comment:
Should remove this once we merge it in. Also, I can't find this comment on the site source, so maybe it didn't work?

Reply:
Since this branch isn't merged to `dev` (or `master`), you won't see it anywhere else besides the local server launched in the `github-actions` branch. I know this is not a good way to check if deployment worked, but I thought a single line of comment won't hurt.

Reply:
If the concern is checking whether or not deploy worked, I think for `dev` we have Heroku logs, and the next time `master` gets updated there'd be a lot of other features added that we can use as indicators? (also GAE logs, but I don't really know where they are lol)

Reply:
If you think it's easier to just leave in the comment for now, just don't forget to remove it once we verify both deployments work!

Reply:
Now that I think about it, this was definitely a dumb mistake that I made when testing GitHub Actions by manually pushing to `master`. Fixed in bfb73944e1775b45e4bdead29d4b82b913f18673.